exdb/tests/browser_tests.py

```diff
@@ -324,3 +313,46 @@ class HallStaffDashboardBrowserTest(DefaultLiveServerTestCase):
     def test_load(self):
         self.client.get(reverse('hallstaff_dash'))
         self.assertEqual(self.driver.find_element(By.XPATH, '//h1').text, _('Experiences Pending Approval'))
+
+
+class CreateExperienceBrowserTest(DefaultLiveServerTestCase):
+
+    def create_spontaneous_type(self):
```

Comment:
Rename this to `setUp` and then you won't need to call it in all the `test_*` methods.